README.md

```diff
@@ -110,13 +111,13 @@ If it is not, you can apply for whitelisting at hello@beamerbridge.com.
 1. For mainnet, run:
 
    ```
-   docker compose -f docker-compose-mainnet.yml up -d
+   docker compose -f docker-compose-mainnet.yml up -d mainnet-agent
```

Comment:
why do we add `mainnet-agent` here?

Reply:
@compojoom ?

Reply:
i can answer as service name is mainnet-agent in the docker-compose if we use this only agent runs and not health-check